### PR TITLE
Tasty Bites ~ Rails Group Project

### PR DESCRIPTION
**Pull request by Lucas-Erkana**
Hi Reviewer with Partner
<hr>
In this PR, **@Lucash2022**  and **@byiringiroscar** have:

- Created a complete project using Rails
- Used Ruby gems as a software package system.
- Installed Ruby on Rails framework.
- Created Rails RESTful design and router.
- Used controllers to handle requests and render empty views.
- Worked with Rails naming conventions.
- Worked with params from browser requests in a safe way.
- Used preprocessed HTML file with embedded Ruby code.
- Used layouts and templates for shared content.
- Used database migration files to maintain database schema.
- Used validations for models.
- Secured our app from n+1 problems.
- Written SQL queries with ActiveRecord.
- Created associations between models.
- Created user login using Devise
- Implemented authorization using CanCanCan.
- Written unit tests using Rspec

Cheers 🥂

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,11 +1,10 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>TasteyBites</title>
+    <title>Tasty Bites</title>
```
Comment (VICTOR-CHUKWU) [inline app/views/layouts/application.html.erb:4]: - [ ] I totally adore your styling, fantastic work guys :+1:

File: app/controllers/foods_controller.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    @food = Food.new(food_params)
+    @food.user = current_user
+
+    respond_to do |format|
+      if @food.save
+        format.html { redirect_to foods_url, notice: 'Food was successfully created.' }
+        format.json { render :index, status: :created, location: @food }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+        format.json { render json: @food.errors, status: :unprocessable_entity }
```
Comment (VICTOR-CHUKWU) [inline app/controllers/foods_controller.rb:32]: - [ ] your redirect works absolutely fine good job :clap:

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Lucash2022  and @byiringiroscar , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed
- [ ] all test passed
- [ ] devise implemented correctly
- [ ] app styled and designed in a fine way
- [ ] app works correctly
- [ ] nice job guys 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (Lucas-Erkana) [reply app/views/layouts/application.html.erb:4]: @VICTOR-CHUKWU 
Thank you so much.

File: app/controllers/foods_controller.rb (hunk 2 above)
Comment (Lucas-Erkana) [reply app/controllers/foods_controller.rb:32]: @VICTOR-CHUKWU 
Thank you so much.

**Pull request merged**